Library/Formula/cassandra-cpp-driver.rb

```diff
@@ -0,0 +1,20 @@
+require "formula"
+
+class CassandraCppDriver < Formula
+  homepage "http://datastax.github.io/cpp-driver/"
+  url "https://github.com/datastax/cpp-driver/archive/1.0.0.tar.gz"
+  sha1 "087aa2cf00e3c6f0eb75bb0471b78d255ea94562"
+  version "1.0.0"
```

Comment:
Unnecessary in this situation :), the Bot will be able to scan this from the URL.